app/services/back_end_service.rb

```diff
@@ -31,8 +31,8 @@ def create_user(user_params)
     def base_url
       # NOTE: base_url needs to be localhost if you want to auth in during development
       # Open your BE server via $ rails s --port 4500
-      # 'http://localhost:4500'
-      'https://spotme-app-api.herokuapp.com'
+      'http://localhost:4500'
+      # 'https://spotme-app-api.herokuapp.com'
```

Comment:
Let's update the `base_url` to be the backend API url before we merge this.

Reply:
I just updated this and rerecorded cassettes.  Please take approve so we can merge